platform/lib/platform_web/components.ex

```diff
@@ -1156,11 +1155,7 @@ defmodule PlatformWeb.Components do
           <dt class="text-sm font-medium text-gray-500 mt-1">Add Attributes</dt>
           <dd class="mt-1 flex flex-wrap gap-2 text-sm text-gray-900 sm:mt-0 sm:col-span-2">
             <%= for attr <- @unset_attrs do %>
-              <%= live_patch("+ #{attr.label}",
-                class: "button original",
-                to: Routes.media_show_path(@socket, :edit, @media.slug, attr.name),
-                replace: true
-              ) %>
+              <.link class = "button original" patch = {Routes.media_show_path(@socket, :edit, @media.slug, attr.name)} replace = {true}></.link>
```

Comment:
This change removes the `"+ #{attr.label}"` value of the button; no label shows in the interface now.

Reply:
<img width="440" alt="image" src="https://github.com/atlosdotorg/atlos/assets/3037552/f448e4d7-f7bb-4b62-b571-da433f90ded4">